src/css/index.scss

```diff
@@ -136,31 +155,35 @@ hr {
 .MatchWidget__suggestion-list {
   display: block;
   margin-top: 3px;
-  padding: 0 $gutter-width $gutter-width $gutter-width;
 }
 
 .MatchWidget__suggestion {
   display: block;
   cursor: pointer;
-  font-weight: 700;
-  font-size: 16px;
+  font-weight: bold;
+  font-size: $font-size-large;
   color: $match-suggestion-color;
+  background-color: $match-suggestion-background-color;
 }
 
 .MatchWidget__suggestion:nth-child(even) {
   background-color: darken($match-suggestion-background-color, 2%);
 }
 
 .MatchWidget__suggestion:hover {
-  background-color: $match-suggestion-color-hover;
+  color: $match-suggestion-color-hover;
+  background-color: $match-suggestion-background-color-hover;
 }
 .MatchWidget__suggestion + .MatchWidget__suggestion {
   margin-top: 3px;
 }
 
 .MatchWidget__feedbackLink {
-  text-align: right;
-  font-size: 12px;
+  margin-top: $gutter-width;
+  font-size: $font-size-x-small;
```

Comment:
This font size is so small it's a bit hard to see.

Reply:
Agreed – shall we bring it up a px, or perhaps just ignore it in favour of `$font-size-small`?

Reply:
Yeah maybe the latter - `$font-size-small`?